tests/test_examples.py

```diff
@@ -236,118 +248,142 @@ def test_run_rbfe_legs(
     mol_b,
     seed,
 ):
-    with temporary_working_dir() as temp_dir:
-        with resources.as_file(resources.files("timemachine.datasets.fep_benchmark.hif2a")) as hif2a_dir:
-            config = dict(
-                mol_a=mol_a,
-                mol_b=mol_b,
-                sdf_path=hif2a_dir / "ligands.sdf",
-                pdb_path=hif2a_dir / "5tbm_prepared.pdb",
-                seed=seed,
-                legs=leg,
-                n_eq_steps=n_eq_steps,
-                n_frames=n_frames,
-                n_windows=n_windows,
-                # Use simple charges to avoid os-dependent charge differences
-                forcefield="smirnoff_1_1_0_sc.py",
+    # Can generate hashes from CI artifacts
```

Comment:
Might be worth documenting the process for updating one of these hashes. E.g. when we intentionally break bitwise reproducibility, can we just copy the new hash printed by the failed CI job here?